docs/troubleshooting-development-issues.md

```diff
@@ -0,0 +1,103 @@
+While facing issues is frustrating, most issues that contributors experience are related to skimming through our docs and skipping important sections and information.
+
```

Comment:
```suggestion
```

I would just remove this. It doesn't seem to be adding anything valuable. It just kind of points the finger at whoever is having issues. Also, the more concise we can make the documentation, the more likely people are to read it.